app/handlers/change_karma.py

```diff
@@ -59,9 +61,34 @@ async def karma_change(message: types.Message, karma: dict, user: User, chat: Ch
         disable_web_page_preview=True,
         reply_markup=kb.get_kb_karma_cancel(user, karma_event)
     )
+    if count_auto_restrict:
+        await message.answer(await render_text_auto_restrict(count_auto_restrict, target))
     asyncio.create_task(remove_kb_after_sleep(msg, config.TIME_TO_CANCEL_ACTIONS))
 
 
+async def render_text_auto_restrict(count_auto_restrict: int, target: User):
+    # TODO чото надо сделать с этим чтобы понятно объяснить за что RO и что будет в следующий раз
+    if it_was_last_one_auto_restrict(count_auto_restrict):
+        about_next = ""
```

Comment:
Don't leave a blank message. Let's figure out what to write.

Reply:
reworked, thanks